README.md

```diff
@@ -73,6 +73,9 @@ When building Swift, keep the following in mind:
 
 To checkout a specific version of Swift, use the `update-checkout` script. For example: `./swift/utils/update-checkout --clone --tag swift-5.0.1-RELEASE`.
 
+_note: The `/update-checkout` script must be ran at the correct tag. e.g. `git clone https://github.com/apple/swift.git --branch swift-5.0.1-RELEASE --depth 1`_
```

Comment:
I think we should move this up, and mention it as an adjustment that needs to be made to the `Swift build instructions`.

I'd also like to update these docs to mention the swift-5.1 tag. If you want to, that would be great.